README.rst

```diff
@@ -140,6 +140,13 @@ Return list of aggregate tuples constructed from elements of multiple iterables.
    >>> dicteval({"=zip": [[1, 2, 3], [4, 5], [6, 7, 8, 9]]})
    [(1, 4, 6), (2, 5, 7)]
 
+Function ``=filter``
+'''''''''''''''''
+
+Returns list of all values that match given function.
+
+   >>> dicteval({"=filter": [check_even, [2, 3, 4]]})
```

Comment:
Hey @osantana , sorry I took a while to get back to you. Would it be possible to explain it once more since I did not completely understand? Is it in a way related to https://github.com/osantana/dicteval/issues/8 ?

Reply:
Hi @Projjol,

The PR #19 has a similar approach to solve this problem. Use it as a reference.

Main issue here is that you are trying to call a Python function instead of a dicteval expression. And when you do it in doctest's README.md it breaks because you didn't imported filter function to test.

Reply:
Hey @osantana ,

Thanks for getting back! I noticed PR #19 also implements a filter function, would this PR serve any purpose given that, that PR is already merged?

Reply:
Hi @Projjol,

Sorry but only now I see that both PR implement the same function.

I think it's better to close this one. Would you like to work in any other?

Reply:
Hey @osantana ,

Yup, I can pick up a different one, thanks! :)